testutil/sample/launch.go

```diff
@@ -207,3 +216,67 @@ func GenesisHash() string {
 	hash := sha256.Sum256([]byte(String(50)))
 	return hex.EncodeToString(hash[:])
 }
+
+// Params returns a sample of params for the launch module
+func Params() launch.Params {
+	maxLaunchTime := rand.Intn(int(launch.MaxParametrableLaunchTime))
+	minLaunchTime := rand.Intn(maxLaunchTime)
+
+	return launch.Params{
+		MinLaunchTime: uint64(minLaunchTime),
+		MaxLaunchTime: uint64(maxLaunchTime),
+	}
+}
+
+// LaunchGenesisState returns a sample genesis state for the launch module
+func LaunchGenesisState() launch.GenesisState {
+	chainID1, _ := ChainID(0)
+	chainID2, _ := ChainID(0)
+
+	return launch.GenesisState{
+		ChainList: []launch.Chain{
+			*Chain(chainID1, Uint64()),
```

Comment:
Does `Chain`, `GenesisAccount` and others actually needs to return pointers in the first place?

Reply:
It's a good point. It used to be more convenient to use pointer since we used them but now we may have some type where it's better to return the value https://github.com/tendermint/spn/issues/254